dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -218,13 +215,14 @@
       <AdditionalOptions>/std:c++17 /bigobj %(AdditionalOptions)</AdditionalOptions>
       <AdditionalOptions>/Wv:18 /Zm1000 %(AdditionalOptions)</AdditionalOptions>
       <AdditionalOptions Condition="'$(Configuration)'=='Release'">%(AdditionalOptions) /d2FH4</AdditionalOptions>
+      <SpectreMitigation Condition="'$(Configuration)'=='Release'">Spectre</SpectreMitigation>
       <WarningLevel>Level4</WarningLevel>
       <TreatWarningAsError>true</TreatWarningAsError>
       <EnablePREfast Condition="'$(Configuration)'=='Release'">true</EnablePREfast>
       <ShowIncludes>false</ShowIncludes>
       <!-- Disable RTTI to keep binary size down (adds about 50% to release dll size) -->
       <RuntimeTypeInfo>false</RuntimeTypeInfo>
-      <PreprocessorDefinitions>%(PreprocessorDefinitions);_HAS_STATIC_RTTI=0</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(Configuration)'=='Release'">%(PreprocessorDefinitions);_HAS_STATIC_RTTI=0</PreprocessorDefinitions>
```

Comment:
Out of curiosity, what does this do?

Reply:
_HAS_STATIC_RTTI was something Ranjesh added to save a little bit of binary size from STL functions so that they don't need RTTI info. I was finding that it was causing an intellisense warning (which I think is bogus) but we don't need this space savings in Debug builds so I restricted the optimization to release.

Reply:
Is there any risk of this causing difference in behavior between debug and release builds ? I'm sure we will catch them though because we run tests in release, so i am not very concerned about it.

Reply:
There is a risk but I feel at the moment that the warning is more annoying than that concern. I think we'll catch this in automation since we do have coverage as you say.